cmd/wallet-web/src/pages/VaultsPage.vue

```diff
@@ -3,6 +3,140 @@
  *
  * SPDX-License-Identifier: Apache-2.0
 -->
+<script>
+import { reactive } from 'vue';
+export const vaultsStore = reactive({
+  vaultsOutdated: false,
+});
+
+export const vaultsMutations = {
+  setVaultsOutdated(value) {
+    vaultsStore.vaultsOutdated = value;
+  },
+};
+</script>
+
+<script setup>
+import { computed, onMounted, ref, watchEffect } from 'vue';
+import { CollectionManager, CredentialManager, WalletUser } from '@trustbloc/wallet-sdk';
+import { mapActions, mapGetters, useStore } from 'vuex';
+import { useI18n } from 'vue-i18n';
+import useBreakpoints from '@/plugins/breakpoints.js';
+import DeleteVaultComponent from '@/components/Vaults/DeleteVaultModalComponent';
+import FlyoutComponent from '@/components/Flyout/FlyoutComponent';
+import FlyoutMenuComponent from '@/components/Flyout/FlyoutMenuComponent';
+import FlyoutButtonComponent from '@/components/Flyout/FlyoutButtonComponent';
+import RenameVaultComponent from '@/components/Vaults/RenameVaultModalComponent';
+import VaultCardComponent from '@/components/Vaults/VaultCardComponent';
+import WelcomeBannerComponent from '@/components/Vaults/VaultWelcomeBannerComponent.vue';
+import SkeletonLoaderComponent from '@/components/SkeletonLoader/SkeletonLoaderComponent.vue';
+
+// Hooks
+const { t } = useI18n();
+const store = useStore();
+
+// Local Variables
+const numOfCreds = ref(0);
+const vaults = ref([]);
+const skippedLocally = ref(false);
+const loading = ref(true);
+const breakpoints = useBreakpoints();
+const showDeleteModal = ref(false);
+const showRenameModal = ref(false);
+const selectedVaultId = ref('');
+const authToken = ref(null);
+const credentialManager = ref(null);
+const collectionManager = ref(null);
+const walletUser = ref(null);
+const skipWelcomeMsg = computed(() => store?.state?.user?.preference?.skipWelcomeMsg);
+const existingNames = computed(() => vaults.value.map((vault) => vault.name));
+
+// Store Getters
+const getCurrentUser = computed(() => store.getters['getCurrentUser']);
+const getAgentInstance = computed(() => store.getters['agent/getInstance']);
```

Comment:
These are not technically functions anymore, so we should just rename them accordingly:
```suggestion
const currentUser = computed(() => store.getters['getCurrentUser']);
const agentInstance = computed(() => store.getters['agent/getInstance']);
```